src/Access/UsersConfigAccessStorage.cpp

```diff
@@ -52,6 +53,38 @@ namespace
 
     UUID generateID(const IAccessEntity & entity) { return generateID(entity.getType(), entity.getName()); }
 
+    template <typename T>
+    void parseGrant(T & entity, const String & string_query)
+    {
+        ParserGrantQuery parser;
+        parser.parseWithoutGrantees();
```

Comment:
This function doesn't grant anything, it just set that the parser shouldn't expect a TO/FROM clause at the end of a statement. I mean the name of the function doesn't look very clear.